src/toil/utils/toilLaunchCluster.py

```diff
@@ -19,6 +19,7 @@
 
 from toil import applianceSelf
 from toil.common import parser_with_common_options
+from toil.lib.aws.utils import build_tag_dict
```

Comment:
It looks like `toil.lib.aws.utils` is designed to be able to import even if the Boto modules (which only come with the `aws` extra) aren't installed. So even if we import this we should still be able to `launch-cluster` if the user is only using Google Cloud and the Google modules.